sky/provision/kubernetes/utils.py

```diff
@@ -1271,3 +1274,18 @@ def check_secret_exists(secret_name: str, namespace: str) -> bool:
         raise
     else:
         return True
+
+
+def get_pod_name(cluster_name_on_cloud: str):
+    """Returns the pod name of the head pod for the given cluster name on cloud
+
+    Args:
+        cluster_name_on_cloud: Name of the cluster on cloud
+
+    Returns:
+        str: Pod name of the head pod
+    """
+    # We could have iterated over all pods in the namespace and checked for the
+    # label, but since we know the naming convention, we can directly return the
+    # head pod name.
+    return f'{cluster_name_on_cloud}-head'
```

Comment:
This seems a very shallow function and is only used once. Should we just have `'{cluster_name_on_cloud}-head'` in the place we call this function?

If we really want to keep this function, it might be better to rename it to `get_head_pod_name` instead, otherwise, it is a bit confusing for the multi-node case

Reply:
Good point - I've refactored to `get_head_pod_name`.